Lib/test/test_peepholer.py

```diff
@@ -2472,6 +2472,13 @@ def test_unoptimized_if_support_killed(self):
         ]
         self.check(insts, insts)
 
+        insts = [
```

Comment:
Please can you include the crasher python code from the issue in a test?

Eg this bit:

```python
def create_obj():
    obj = [42]
    try:
        return obj
    finally:
        del obj

self.assertListEqual(create_obj(), [42])
```

Reply:
I personally can't repro this crash. But I haven't tried putting it into a test method.